src/features/pageObjects/styles/pageObject.less

```diff
@@ -32,13 +31,34 @@
     .ant-collapse-header {
       gap: 8px;
 
+      .ant-collapse-header-text {
+        display: flex;
+      }
+
+      .ant-collapse-extra {
+        display: flex;
+
+        .ant-dropdown {
+          left: -120px !important; // fixed context menu under the button
+        }
+      }
+
+      .jdn__locatorsList_button-copy,
+      .jdn__locatorsList_button-menu {

```

Comment:
I think you can rename these styles like `.jdn__pageObject.....`, since the don't nest any original locator's styles anymore